src/dialogs/quick-open/show-dialog-quick-open.ts

```diff
@@ -0,0 +1,20 @@
+import { fireEvent } from "../../common/dom/fire_event";
+
+export interface QuickOpenDialogParams {
+  entityFilter?: string;
```

Comment:
Do we use this? How would we use this?

Reply:
Not needed anymore. It now lives in show-dialog-quick-open.ts. Will remove.